### DISCUSSION TITLE
Could not resolve com.arkivanov.decompose:decompose:1.0.0-alpha-05

### DISCUSSION THREAD

**Discussion by mchakkarwar** [arkivanov/Decompose#281] [General]
Hi @arkivanov 
I have to use decompose library in kotlin multiplatform. For keeping navigation logic common, I an trying to add decompose library in common module. But, I am getting below error;
:common:commonMain: Could not resolve com.arkivanov.decompose:decompose:1.0.0-alpha-05.
I am  trying to add dependency as below;
  val commonMain by getting {
            dependencies {
                api(compose.runtime)
                api(compose.foundation)
                api(compose.material)
                implementation("com.arkivanov.decompose:decompose:1.0.0-alpha-05")
            }
        }
Please help me, I am blocking from long time.

**Comment by arkivanov** [comment]
Hi, could you please attach a complete build log? And your build.gradle file please.

**Comment by mchakkarwar** [comment]
Hi @arkivanov 
please find complete build.gradle.kts file.
[build.gradle.kts.zip](https://github.com/arkivanov/Decompose/files/10177306/build.gradle.kts.zip)

**Reply by arkivanov** [reply]
Thanks, could you please also provide a complete build log?

**Comment by mchakkarwar** [comment]
Hi @arkivanov ,
Please find build screenshot, on syncing gradle after adding dependency getting below error;
<img width="1363" alt="Screenshot 2022-12-07 at 8 50 51 PM" src="https://user-images.githubusercontent.com/2756652/206218645-818a7484-6675-4450-a4cd-91b1e0f10209.png">

**Reply by arkivanov** [reply]
Sorry, could you please attach the full log? Not just the error message? You can try `./gradlew :common:build > log.txt`.

**Reply by arkivanov** [reply]
Alternatively, you could attach a reproducer project.

**Reply by mchakkarwar** [reply]
Ok..I will share logs

**Reply by mchakkarwar** [reply]
@arkivanov Please find logs.txt file for logs
[log.txt.](https://github.com/arkivanov/Decompose/files/10177471/log.txt)

**Reply by mchakkarwar** [reply]
Thanks @arkivanov, issue has been solved by adding sdk path in local.properties.

**Reply by arkivanov** [reply]
Glad you solved the issue!

**Comment by mchakkarwar** [comment]
Hi @arkivanov 
After adding dependency  implementation("com.arkivanov.decompose:decompose:1.0.0-alpha-05") in commonMain. Still dependencies not resolving. Not getting Component and other classes dependency.

Here is my build.gradle file for common module.
 
[build.gradle.kts.zip](https://github.com/arkivanov/Decompose/files/10187753/build.gradle.kts.zip)

**Reply by arkivanov** [reply]
Would you be able to provide a reproducer project?

**Reply by mchakkarwar** [reply]
How to do it?

**Reply by mchakkarwar** [reply]
Here I am adding Component and other files
<img width="378" alt="Screenshot 2022-12-08 at 10 43 30 PM" src="https://user-images.githubusercontent.com/2756652/206519280-9c154701-aa0d-47c7-823a-e83457e9077e.png">

**Reply by arkivanov** [reply]
You could just zip your project folder and attach it here, if possible. It would simplify the investigation a lot.

**Reply by mchakkarwar** [reply]
Ok

**Reply by mchakkarwar** [reply]
Please find project source code and dependencies of common module;

[LoginDemo.zip](https://github.com/arkivanov/Decompose/files/10187858/LoginDemo.zip)
[dependencies.txt](https://github.com/arkivanov/Decompose/files/10187862/dependencies.txt)

**Reply by arkivanov** [reply]
Thanks! Basically, you are using old versions of Kotlin and Compose. I recommend using the following versions:

- Kotlin `1.7.20`
- JetBrains Compose `1.2.1`
- Decompose `1.0.0-beta-01`

**Reply by mchakkarwar** [reply]
Thank you @arkivanov, it is working after changing version.

**Comment by mchakkarwar** [comment]
Thank you @arkivanov for quick support, I have successfully delivered demo in my company on Kotlin Multiplatform.

**Reply by arkivanov** [reply]
Great news! Thanks for using there library!
